packages/pug/test/anti-cases/Invalid-Attribute-test/invalid.test.js

```diff
@@ -0,0 +1,12 @@
+const pug = require('../../../');
+
+test('invalid attribute name should log warning', () => {
+  const oldWarn = console.warn;
+  const warnings = [];
+  console.warn = warnings.push.bind(warnings);
+  pug.compileFile(
+    __dirname + '/invalid-character.pug'
+  );
+  console.warn = oldWarn;
+  expect(warnings).toMatchSnapshot();
```

Comment:
Here you should use something like `warnings.map(warning => warning.replace(/\\/g, '/').split(process.cwd().replace(/\\/g, '/')).join('<cwd>'))` to normalise the file path and remove the current working directory prefix. This should make the tests pass both locally and on travis.